test/chunking-form/samples/chunking-compact/_expected/amd/chunk-1e9855f7.js

```diff
@@ -0,0 +1,6 @@
+define(['exports'],function(exports){'use strict';function fn () {
+  console.log('lib2 fn');
+}function fn$1 () {
```

Comment:
This only works because function declarations do not require semicolons to be on the same line as other statements. The same does not hold for e.g. other declarations—
```javascript
const x =3function y(){}
```
would NOT be valid code, even if you add a space in between. A semicolon would be the proper solution here.

Reply:
Note that in the `const x = 3` case, the VariableDeclaration render function will ensure a trailing semicolon already for us.

Reply:
Oh, you are right! For declarations and expression statements, this is taken care of 👍